taxjar_salestax/models/taxjar_config.py

```diff
@@ -212,6 +216,17 @@ def create_order(
             _logger.debug('Response: %s', res)
         return res
 
+    @api.model
+    def _get_discount_percentage_from_order_item(self, item):
```

Comment:
I believe it should be `_get_discount_amount_from_order_item` and `discount_amount`.